app/views/budgets/investments/_comments.html.erb

```diff
@@ -0,0 +1,41 @@
+<% commentable = comment_tree.commentable %>
+<% current_order = comment_tree.order %>
+
+<% cache [locale_and_user_status, current_order, commentable_cache_key(commentable), comment_tree.comments, comment_tree.comment_authors, commentable.comments_count, comment_flags] do %>
```

Comment:
Allright! Realized this is as well on budgets and comments partials... we'll deal with all once this goes into master 👍